packages/context/src/context-view.ts

```diff
@@ -151,6 +151,19 @@ export class ContextView<T = unknown> extends EventEmitter
   asGetter(session?: ResolutionSession): Getter<T[]> {
     return () => this.values(session);
   }
+
+  /**
+   * Get the single value
+   */
+  async singleValue(session?: ResolutionSession): Promise<T | undefined> {
+    const values = await this.values(session);
+    if (values.length === 0) return undefined;
+    if (values.length === 1) return values[0];
+    else
```

Comment:
`else` is not needed here, previous branches always return from the function. Please remove this line, it will make the code less indented and probably easier to read too.

Reply:
> `else` is not needed here, previous branches always return from the function. Please remove this line, it will make the code less indented and probably easier to read too.

☝️